File: Code/c/pattern1.c
#include<stdio.h>
void main()
{
    int limit, i, j;

    printf("Enter Limit: ");
    scanf("%d", &limit);

    for (i = 1; i <= limit; i++)
    {
        for (j = 1; j <= i; j++)
            printf("* ");
        printf("\n");
    }
}

File: Code/java/pattern1.java
import java.util.Scanner;

class Pattern1
{
    public static void main(String[] args)
    {
        int limit, i;
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Limit: ");
        limit = scanner.nextInt();

        for (i = 1; i <= limit; i++)
            System.out.println("* ".repeat(i));

        scanner.close();
    }
}